commands/account/clean.ts

```diff
@@ -51,10 +54,10 @@ exports.handler = async options => {
 
   for (const account of filteredTestAccounts) {
     try {
-      await accessTokenForPersonalAccessKey(
-        getAccountIdentifier(account),
-        true
-      );
+      const accountId = getAccountIdentifier(account);
+      if (accountId) {
```

Comment:
I would just double check that this doesn't change any existing behavior - previously if there was no accountId, the catch block would be triggered

Reply:
Oh good call, I'll take a look to make sure

Reply:
Fixed this in my latest commit. I think it did change the behavior